### PR TITLE
You can now unselect the groups for the student

### PR DESCRIPTION
**Pull request by a17adasy**
image

If you unselect all boxes it will now say none, before you couldnt unselect all groups for a student.

### CODE REVIEW

**Comment by a17adasy** [comment]
I see now that my editor har changed spaces and what not so it looks like hundreds of lines have changed.
The line that was added is on line number 433 in accessed.js.

**Comment by b17timjo** [comment]
Not that any code in the project really adheres to the code standard, but when changing or adding, one might as well keep it in mind, so...
I'm gonna try a feature github pointed out and suggest changes to improve code standard -ness

File: DuggaSys/accessed.js [hunk 1]
```diff
@@ -13,226 +13,208 @@ var myTable;
 //----------==========########## User Interface ##########==========----------
 //----------------------------------------------------------------------------
 
-function setup()
-{
-  AJAXService("GET",{cid:querystring['cid'],coursevers:querystring['coursevers']},"ACCESS");
+function setup() {
```
Comment (b17timjo) [inline DuggaSys/accessed.js]:
```suggestion
function setup()
{
```

File: DuggaSys/accessed.js [hunk 2]
```diff
@@ -13,226 +13,208 @@ var myTable;
...
 
 //  Instead of commenting out the functions as previously which caused uncaught reference errors
 //  function content was commented out to avoid having a white empty box appear.
-function hoverc()
-{
-    /*
-    $('#dropdowns').css('display','none');
-    $('#dropdownc').css('display','block');
-    */
+function hoverc() {
```
Comment (b17timjo) [inline DuggaSys/accessed.js]:
```suggestion
function hoverc()
{
```

File: DuggaSys/accessed.js [hunk 3]
```diff
@@ -13,226 +13,208 @@ var myTable;
...
+	/*
+	$('#dropdowns').css('display','none');
+	$('#dropdownc').css('display','block');
+	*/
 }
 
-function hovers()
-{
+function hovers() {
```
Comment (b17timjo) [inline DuggaSys/accessed.js]:
```suggestion
function hovers()
{
```

File: DuggaSys/accessed.js [hunk 4]
```diff
@@ -13,226 +13,208 @@ var myTable;
...
   /*
   $('#dropdowns').css('display','block');
   $('#dropdownc').css('display','none');
   */
 }
 
-function leavec()
-{
-		$('#dropdownc').css('display','none');
+function leavec() {
```
Comment (b17timjo) [inline DuggaSys/accessed.js]:
```suggestion
function leavec()
{
```

File: DuggaSys/accessed.js [hunk 5]
```diff
@@ -13,226 +13,208 @@ var myTable;
...
+	$('#dropdownc').css('display', 'none');
 }
 
-function leaves()
-{
-		$('#dropdowns').css('display','none');
+function leaves() {
```
Comment (b17timjo) [inline DuggaSys/accessed.js]:
```suggestion
function leaves()
{
```

File: DuggaSys/accessed.js [hunk 6]
```diff
@@ -13,226 +13,208 @@ var myTable;
...
+	$('#dropdowns').css('display', 'none');
 }
 
-function showCreateUserPopup()
-{
-	$("#createUser").css("display","flex");
+function showCreateUserPopup() {
```
Comment (b17timjo) [inline DuggaSys/accessed.js]:
```suggestion
function showCreateUserPopup()
{
```

File: DuggaSys/accessed.js [hunk 7]
```diff
@@ -13,226 +13,208 @@ var myTable;
...
+	$("#createUser").css("display", "flex");
 }
 
-function showCreateClassPopup()
-{
-	$("#createClass").css("display","flex");
+function showCreateClassPopup() {
```
Comment (b17timjo) [inline DuggaSys/accessed.js]:
```suggestion
function showCreateClassPopup()
{
```

File: DuggaSys/accessed.js [hunk 8]
```diff
@@ -13,226 +13,208 @@ var myTable;
...
+	$("#createClass").css("display", "flex");
 }
 
-function showImportUsersPopup()
-{
+function showImportUsersPopup() {
```
Comment (b17timjo) [inline DuggaSys/accessed.js]:
```suggestion
function showImportUsersPopup()
{
```

File: DuggaSys/accessed.js [hunk 9]
```diff
@@ -13,226 +13,208 @@ var myTable;
...
 	$("#importUsers").css("display", "flex");
 }
 
-function hideCreateUserPopup()
-{
-    $("#createUser").css("display","none");
+function hideCreateUserPopup() {
```
Comment (b17timjo) [inline DuggaSys/accessed.js]:
```suggestion
function hideCreateUserPopup()
{
```

File: DuggaSys/accessed.js [hunk 10]
```diff
@@ -13,226 +13,208 @@ var myTable;
...
+	$("#createUser").css("display", "none");
 }
 
-function hideCreateClassPopup()
-{
-    $("#createClass").css("display","none");
+function hideCreateClassPopup() {
```
Comment (b17timjo) [inline DuggaSys/accessed.js]:
```suggestion
function hideCreateClassPopup()
{
```

File: DuggaSys/accessed.js [hunk 11]
```diff
@@ -13,226 +13,208 @@ var myTable;
...
+	$("#createClass").css("display", "none");
 }
 
-function hideImportUsersPopup()
-{
-	$("#importUsers").css("display","none");
+function hideImportUsersPopup() {
```
Comment (b17timjo) [inline DuggaSys/accessed.js]:
```suggestion
function hideImportUsersPopup()
{
```

File: DuggaSys/accessed.js [hunk 12]
```diff
@@ -13,226 +13,208 @@ var myTable;
...
+	$("#importUsers").css("display", "none");
 }
 
-function closeEdituser()
-{
-    $("#editUsers").css("display","none");
+function closeEdituser() {
```
Comment (b17timjo) [inline DuggaSys/accessed.js]:
```suggestion
function closeEdituser()
{
```

File: DuggaSys/accessed.js [hunk 13]
```diff
@@ -13,226 +13,208 @@ var myTable;
...
+	$("#editUsers").css("display", "none");
 }
 
 //----------------------------------------------------------------------------
 //-------------==========########## Commands ##########==========-------------
 //----------------------------------------------------------------------------
 
-function importUsers()
-{
+function importUsers() {
```
Comment (b17timjo) [inline DuggaSys/accessed.js]:
```suggestion
function importUsers()
{
```

File: DuggaSys/accessed.js [hunk 14]
```diff
@@ -13,226 +13,208 @@ var myTable;
...
-    newUser.push($("#addPid").val() + ', ' + $("#addTerm").val());
-    newUser.push($("#addEmail").val());
-
-    var outerArr = new Array();
-    outerArr.push(newUser);
-
-    var newUserJSON = JSON.stringify(outerArr);
-    AJAXService("ADDUSR",{cid:querystring['cid'],newusers:newUserJSON,coursevers:querystring['coursevers']},"ACCESS");
-    hideCreateUserPopup();
+function addSingleUser() {
```
Comment (b17timjo) [inline DuggaSys/accessed.js]:
```suggestion
function addSingleUser()
{
```

File: DuggaSys/accessed.js [hunk 15]
```diff
@@ -13,226 +13,208 @@ var myTable;
...
-
-  if(inputVerified){
-    var outerArr = new Array();
-    outerArr.push(newClass);
-
-    var newClassJSON = JSON.stringify(outerArr);
-    AJAXService("ADDCLASS",{cid:querystring['cid'],newclass:newClassJSON,coursevers:querystring['coursevers']},"ACCESS");
-    hideCreateClassPopup();
-  }
+function addClass() {
```
Comment (b17timjo) [inline DuggaSys/accessed.js]:
```suggestion
function addClass()
{
```

File: DuggaSys/accessed.js [hunk 16]
```diff
@@ -13,226 +13,208 @@ var myTable;
...
+		var newClassJSON = JSON.stringify(outerArr);
+		AJAXService("ADDCLASS", { cid: querystring['cid'], newclass: newClassJSON, coursevers: querystring['coursevers'] }, "ACCESS");
+		hideCreateClassPopup();
+	}
 }
 
-function resetPw(uid,username)
-{
-    rnd=randomstring();
+function resetPw(uid, username) {
```
Comment (b17timjo) [inline DuggaSys/accessed.js]:
```suggestion
function resetPw(uid, username)
{
```

File: DuggaSys/accessed.js [hunk 17]
```diff
@@ -13,226 +13,208 @@ var myTable;
...
 
-    AJAXService("CHPWD",{cid:querystring['cid'],uid:uid,pw:rnd,coursevers:querystring['coursevers']},"ACCESS");
+	AJAXService("CHPWD", { cid: querystring['cid'], uid: uid, pw: rnd, coursevers: querystring['coursevers'] }, "ACCESS");
 }
 
-function changeOpt(e)
-{
-		var paramlist=e.target.id.split("_");
-		changeProperty(paramlist[1],paramlist[0],e.target.value);
+function changeOpt(e) {
```
Comment (b17timjo) [inline DuggaSys/accessed.js]:
```suggestion
function changeOpt(e)
{
```

File: DuggaSys/accessed.js [hunk 18]
```diff
@@ -13,226 +13,208 @@ var myTable;
...
+	var paramlist = e.target.id.split("_");
+	changeProperty(paramlist[1], paramlist[0], e.target.value);
 }
 
-function changeProperty(targetobj,propertyname,propertyvalue)
-{
-		AJAXService("UPDATE",{cid:querystring['cid'],uid:targetobj,prop:propertyname,val:propertyvalue},"ACCESS");
+function changeProperty(targetobj, propertyname, propertyvalue) {
```
Comment (b17timjo) [inline DuggaSys/accessed.js]:
```suggestion
function changeProperty(targetobj, propertyname, propertyvalue)
{
```

File: DuggaSys/accessed.js [hunk 19]
```diff
@@ -419,41 +401,45 @@ $(document).on("touchend", function (e) {
...
 
 //----------------------------------------------------------------------------------
 // createQuickItem: Handle "fast" click on FAB button
 //----------------------------------------------------------------------------------
 
-function createQuickItem()
-{
-		clearTimeout(pressTimer);
-		showImportUsersPopup();
+function createQuickItem() {
```
Comment (b17timjo) [inline DuggaSys/accessed.js]:
```suggestion
function createQuickItem()
{
```

**Comment by a17adasy** [comment]
Approved all changes cause they all were following code standard better than previously.

File: DuggaSys/accessed.js [hunk 20]
```diff
@@ -15,17 +15,17 @@ var myTable;
 
 function setup()
 {
-  AJAXService("GET",{cid:querystring['cid'],coursevers:querystring['coursevers']},"ACCESS");
+	AJAXService("GET", { cid: querystring['cid'], coursevers: querystring['coursevers'] }, "ACCESS");
```
Comment (b17timjo) [inline DuggaSys/accessed.js]:
```suggestion
		AJAXService("GET", { cid: querystring['cid'], coursevers: querystring['coursevers'] }, "ACCESS");
```

File: DuggaSys/accessed.js [hunk 21]
```diff
@@ -38,22 +38,22 @@ function hovers()
 
 function leavec()
 {
-		$('#dropdownc').css('display','none');
+	$('#dropdownc').css('display', 'none');
```
Comment (b17timjo) [inline DuggaSys/accessed.js]:
```suggestion
		$('#dropdownc').css('display', 'none');
```

File: DuggaSys/accessed.js [hunk 22]
```diff
@@ -38,22 +38,22 @@ function hovers()
...
 }
 
 function leaves()
 {
-		$('#dropdowns').css('display','none');
+	$('#dropdowns').css('display', 'none');
```
Comment (b17timjo) [inline DuggaSys/accessed.js]:
```suggestion
		$('#dropdowns').css('display', 'none');
```

File: DuggaSys/accessed.js [hunk 23]
```diff
@@ -38,22 +38,22 @@ function hovers()
...
 }
 
 function showCreateUserPopup()
 {
-	$("#createUser").css("display","flex");
+	$("#createUser").css("display", "flex");
```
Comment (b17timjo) [inline DuggaSys/accessed.js]:
```suggestion
		$("#createUser").css("display", "flex");
```

File: DuggaSys/accessed.js [hunk 24]
```diff
@@ -38,22 +38,22 @@ function hovers()
...
 }
 
 function showCreateClassPopup()
 {
-	$("#createClass").css("display","flex");
+	$("#createClass").css("display", "flex");
```
Comment (b17timjo) [inline DuggaSys/accessed.js]:
```suggestion
		$("#createClass").css("display", "flex");
```

File: DuggaSys/accessed.js [hunk 25]
```diff
@@ -63,22 +63,22 @@ function showImportUsersPopup()
...
 function hideImportUsersPopup()
 {
-	$("#importUsers").css("display","none");
+	$("#importUsers").css("display", "none");
 }
 
 function closeEdituser()
 {
-    $("#editUsers").css("display","none");
+	$("#editUsers").css("display", "none");
```
Comment (b17timjo) [inline DuggaSys/accessed.js]:
```suggestion
		$("#editUsers").css("display", "none");
```

File: DuggaSys/accessed.js [hunk 26]
```diff
@@ -63,22 +63,22 @@ function showImportUsersPopup()
...
 function hideCreateClassPopup()
 {
-    $("#createClass").css("display","none");
+	$("#createClass").css("display", "none");
 }
 
 function hideImportUsersPopup()
 {
-	$("#importUsers").css("display","none");
+	$("#importUsers").css("display", "none");
```
Comment (b17timjo) [inline DuggaSys/accessed.js]:
```suggestion
		$("#importUsers").css("display", "none");
```

File: DuggaSys/accessed.js [hunk 27]
```diff
@@ -63,22 +63,22 @@ function showImportUsersPopup()
...
 function hideCreateUserPopup()
 {
-    $("#createUser").css("display","none");
+	$("#createUser").css("display", "none");
 }
 
 function hideCreateClassPopup()
 {
-    $("#createClass").css("display","none");
+	$("#createClass").css("display", "none");
```
Comment (b17timjo) [inline DuggaSys/accessed.js]:
```suggestion
		$("#createClass").css("display", "none");
```

File: DuggaSys/accessed.js [hunk 28]
```diff
@@ -63,22 +63,22 @@ function showImportUsersPopup()
 
 function hideCreateUserPopup()
 {
-    $("#createUser").css("display","none");
+	$("#createUser").css("display", "none");
```
Comment (b17timjo) [inline DuggaSys/accessed.js]:
```suggestion
		$("#createUser").css("display", "none");
```

File: DuggaSys/accessed.js [hunk 29]
```diff
@@ -88,151 +88,151 @@ function closeEdituser()
...
-    newUser.push($("#addPid").val() + ', ' + $("#addTerm").val());
-    newUser.push($("#addEmail").val());
-
-    var outerArr = new Array();
-    outerArr.push(newUser);
-
-    var newUserJSON = JSON.stringify(outerArr);
-    AJAXService("ADDUSR",{cid:querystring['cid'],newusers:newUserJSON,coursevers:querystring['coursevers']},"ACCESS");
-    hideCreateUserPopup();
+	var newUser = new Array();
```
Comment (b17timjo) [inline DuggaSys/accessed.js]:
```suggestion
		var newUser = new Array();
```

File: DuggaSys/accessed.js [hunk 30]
```diff
@@ -88,151 +88,151 @@ function closeEdituser()
...
+	newUser.push($("#addSsn").val());
```
Comment (b17timjo) [inline DuggaSys/accessed.js]:
```suggestion
		newUser.push($("#addSsn").val());
```

File: DuggaSys/accessed.js [hunk 31]
```diff
@@ -88,151 +88,151 @@ function closeEdituser()
...
+	newUser.push($("#addLastname").val() + ", " + $("#addFirstname").val());
```
Comment (b17timjo) [inline DuggaSys/accessed.js]:
```suggestion
		newUser.push($("#addLastname").val() + ", " + $("#addFirstname").val());
```

File: DuggaSys/accessed.js [hunk 32]
```diff
@@ -88,151 +88,151 @@ function closeEdituser()
...
+	newUser.push($("#addCid").val());
```
Comment (b17timjo) [inline DuggaSys/accessed.js]:
```suggestion
		newUser.push($("#addCid").val());
```

File: DuggaSys/accessed.js [hunk 33]
```diff
@@ -88,151 +88,151 @@ function closeEdituser()
...
+	newUser.push($("#addNy").val());
```
Comment (b17timjo) [inline DuggaSys/accessed.js]:
```suggestion
		newUser.push($("#addNy").val());
```

File: DuggaSys/accessed.js [hunk 34]
```diff
@@ -88,151 +88,151 @@ function closeEdituser()
...
+	newUser.push($("#addPid").val() + ', ' + $("#addTerm").val());
```
Comment (b17timjo) [inline DuggaSys/accessed.js]:
```suggestion
		newUser.push($("#addPid").val() + ', ' + $("#addTerm").val());
```

File: DuggaSys/accessed.js [hunk 35]
```diff
@@ -88,151 +88,151 @@ function closeEdituser()
...
+	newUser.push($("#addEmail").val());
```
Comment (b17timjo) [inline DuggaSys/accessed.js]:
```suggestion
		newUser.push($("#addEmail").val());
```

File: DuggaSys/accessed.js [hunk 36]
```diff
@@ -88,151 +88,151 @@ function closeEdituser()
...
+
+	var outerArr = new Array();
```
Comment (b17timjo) [inline DuggaSys/accessed.js]:
```suggestion
		var outerArr = new Array();
```

File: DuggaSys/accessed.js [hunk 37]
```diff
@@ -88,151 +88,151 @@ function closeEdituser()
...
+	outerArr.push(newUser);
```
Comment (b17timjo) [inline DuggaSys/accessed.js]:
```suggestion
		outerArr.push(newUser);
```

File: DuggaSys/accessed.js [hunk 38]
```diff
@@ -88,151 +88,151 @@ function closeEdituser()
...
+
+	var newUserJSON = JSON.stringify(outerArr);
```
Comment (b17timjo) [inline DuggaSys/accessed.js]:
```suggestion
		var newUserJSON = JSON.stringify(outerArr);
```

File: DuggaSys/accessed.js [hunk 39]
```diff
@@ -88,151 +88,151 @@ function closeEdituser()
...
+	AJAXService("ADDUSR", { cid: querystring['cid'], newusers: newUserJSON, coursevers: querystring['coursevers'] }, "ACCESS");
```
Comment (b17timjo) [inline DuggaSys/accessed.js]:
```suggestion
		AJAXService("ADDUSR", { cid: querystring['cid'], newusers: newUserJSON, coursevers: querystring['coursevers'] }, "ACCESS");
```

File: DuggaSys/accessed.js [hunk 40]
```diff
@@ -88,151 +88,151 @@ function closeEdituser()
...
+	hideCreateUserPopup();
```
Comment (b17timjo) [inline DuggaSys/accessed.js]:
```suggestion
		hideCreateUserPopup();
```

File: DuggaSys/accessed.js [hunk 41]
```diff
@@ -88,151 +88,151 @@ function closeEdituser()
...
-
-  if(inputVerified){
-    var outerArr = new Array();
-    outerArr.push(newClass);
-
-    var newClassJSON = JSON.stringify(outerArr);
-    AJAXService("ADDCLASS",{cid:querystring['cid'],newclass:newClassJSON,coursevers:querystring['coursevers']},"ACCESS");
-    hideCreateClassPopup();
-  }
+	inputVerified = true;
```
Comment (b17timjo) [inline DuggaSys/accessed.js]:
```suggestion
		inputVerified = true;
```

File: DuggaSys/accessed.js [hunk 42]
```diff
@@ -88,151 +88,151 @@ function closeEdituser()
...
+	document.getElementById("classErrorText").innerHTML = "";
```
Comment (b17timjo) [inline DuggaSys/accessed.js]:
```suggestion
		document.getElementById("classErrorText").innerHTML = "";
```

File: DuggaSys/accessed.js [hunk 43]
```diff
@@ -88,151 +88,151 @@ function closeEdituser()
...
+	var newClass = new Array();
```
Comment (b17timjo) [inline DuggaSys/accessed.js]:
```suggestion
		var newClass = new Array();
```

File: DuggaSys/accessed.js [hunk 44]
```diff
@@ -88,151 +88,151 @@ function closeEdituser()
...
+	newClass.push(verifyClassInput($("#addClass"), null, ""));
```
Comment (b17timjo) [inline DuggaSys/accessed.js]:
```suggestion
		newClass.push(verifyClassInput($("#addClass"), null, ""));
```

File: DuggaSys/accessed.js [hunk 45]
```diff
@@ -88,151 +88,151 @@ function closeEdituser()
...
+	newClass.push(verifyClassInput($("#addResponsible"), null, ""));
```
Comment (b17timjo) [inline DuggaSys/accessed.js]:
```suggestion
		newClass.push(verifyClassInput($("#addResponsible"), null, ""));
```

File: DuggaSys/accessed.js [hunk 46]
```diff
@@ -88,151 +88,151 @@ function closeEdituser()
...
+	newClass.push(verifyClassInput($("#addClassname"), null, ""));
```
Comment (b17timjo) [inline DuggaSys/accessed.js]:
```suggestion
		newClass.push(verifyClassInput($("#addClassname"), null, ""));
```

File: DuggaSys/accessed.js [hunk 47]
```diff
@@ -88,151 +88,151 @@ function closeEdituser()
...
+	newClass.push(verifyClassInput($("#addRegcode"), /^[0-9]*$/, "number"));
```
Comment (b17timjo) [inline DuggaSys/accessed.js]:
```suggestion
		newClass.push(verifyClassInput($("#addRegcode"), /^[0-9]*$/, "number"));
```

File: DuggaSys/accessed.js [hunk 48]
```diff
@@ -88,151 +88,151 @@ function closeEdituser()
...
+	newClass.push(verifyClassInput($("#addClasscode"), null, ""));
```
Comment (b17timjo) [inline DuggaSys/accessed.js]:
```suggestion
		newClass.push(verifyClassInput($("#addClasscode"), null, ""));
```

File: DuggaSys/accessed.js [hunk 49]
```diff
@@ -88,151 +88,151 @@ function closeEdituser()
...
+	newClass.push(verifyClassInput($("#addHp"), /^[0-9.]*$/, "(decimal) number"));
```
Comment (b17timjo) [inline DuggaSys/accessed.js]:
```suggestion
		newClass.push(verifyClassInput($("#addHp"), /^[0-9.]*$/, "(decimal) number"));
```

File: DuggaSys/accessed.js [hunk 50]
```diff
@@ -88,151 +88,151 @@ function closeEdituser()
...
+	newClass.push(verifyClassInput($("#addTempo"), /^[0-9]*$/, "number"));
```
Comment (b17timjo) [inline DuggaSys/accessed.js]:
```suggestion
		newClass.push(verifyClassInput($("#addTempo"), /^[0-9]*$/, "number"));
```

File: DuggaSys/accessed.js [hunk 51]
```diff
@@ -88,151 +88,151 @@ function closeEdituser()
...
+	newClass.push(verifyClassInput($("#addHpProgress"), /^[0-9.]*$/, "(decimal) number"));
```
Comment (b17timjo) [inline DuggaSys/accessed.js]:
```suggestion
		newClass.push(verifyClassInput($("#addHpProgress"), /^[0-9.]*$/, "(decimal) number"));
```

File: DuggaSys/accessed.js [hunk 52]
```diff
@@ -88,151 +88,151 @@ function closeEdituser()
...
+
+	if (inputVerified) {
```
Comment (b17timjo) [inline DuggaSys/accessed.js]:
```suggestion
		if (inputVerified) {
```

File: DuggaSys/accessed.js [hunk 53]
```diff
@@ -88,151 +88,151 @@ function closeEdituser()
...
+		var outerArr = new Array();
```
Comment (b17timjo) [inline DuggaSys/accessed.js]:
```suggestion
				var outerArr = new Array();
```

File: DuggaSys/accessed.js [hunk 54]
```diff
@@ -88,151 +88,151 @@ function closeEdituser()
...
+		outerArr.push(newClass);
```
Comment (b17timjo) [inline DuggaSys/accessed.js]:
```suggestion
				outerArr.push(newClass);
```

File: DuggaSys/accessed.js [hunk 55]
```diff
@@ -88,151 +88,151 @@ function closeEdituser()
...
+
+		var newClassJSON = JSON.stringify(outerArr);
```
Comment (b17timjo) [inline DuggaSys/accessed.js]:
```suggestion
				var newClassJSON = JSON.stringify(outerArr);
```

File: DuggaSys/accessed.js [hunk 56]
```diff
@@ -88,151 +88,151 @@ function closeEdituser()
...
+		AJAXService("ADDCLASS", { cid: querystring['cid'], newclass: newClassJSON, coursevers: querystring['coursevers'] }, "ACCESS");
```
Comment (b17timjo) [inline DuggaSys/accessed.js]:
```suggestion
				AJAXService("ADDCLASS", { cid: querystring['cid'], newclass: newClassJSON, coursevers: querystring['coursevers'] }, "ACCESS");
```

File: DuggaSys/accessed.js [hunk 57]
```diff
@@ -88,151 +88,151 @@ function closeEdituser()
...
+		hideCreateClassPopup();
```
Comment (b17timjo) [inline DuggaSys/accessed.js]:
```suggestion
				hideCreateClassPopup();
```

File: DuggaSys/accessed.js [hunk 58]
```diff
@@ -88,151 +88,151 @@ function closeEdituser()
...
+	}
```
Comment (b17timjo) [inline DuggaSys/accessed.js]:
```suggestion
		}
```

File: DuggaSys/accessed.js [hunk 59]
```diff
@@ -419,33 +419,38 @@ $(document).on("touchend", function (e) {
...
+function mouseUp(e) {
 
 	// if the target of the click isn't the container nor a descendant of the container
 	if (activeElement) {
 		var checkboxes = $(activeElement).find(".checkboxes");
 		checkboxes = activeElement.parentElement.lastChild;
 
-		if (expanded && !checkboxes.contains(e.target)){
+		if (checkboxes.contains(e.target)) {
+			console.log("nämen okej");
```
Comment (b17timjo) [inline DuggaSys/accessed.js]: Don't add logging to console outside personal development
```suggestion
```

**Comment by a17adasy** [comment]
All suggested changes are implemented now.

File: DuggaSys/accessed.js [hunk 60]
```diff
@@ -419,33 +419,37 @@ $(document).on("touchend", function (e) {
...
-function mouseUp(e){
+function mouseUp(e) {
 
 	// if the target of the click isn't the container nor a descendant of the container
 	if (activeElement) {
 		var checkboxes = $(activeElement).find(".checkboxes");
 		checkboxes = activeElement.parentElement.lastChild;
 
-		if (expanded && !checkboxes.contains(e.target)){
+		if (checkboxes.contains(e.target)) {
```
Comment (b17timjo) [inline DuggaSys/accessed.js]:
```suggestion
```

File: DuggaSys/accessed.js [hunk 61]
```diff
@@ -419,33 +419,37 @@ $(document).on("touchend", function (e) {
...
+		}
```
Comment (b17timjo) [inline DuggaSys/accessed.js]:
```suggestion
```

**Pull request merged**